pet-battle/stage/values.yaml

```diff
@@ -0,0 +1,10 @@
+release: "stage"
```

Comment:
hmm it seems like we have `staging-app-of-pb` in [values.yaml](https://github.com/springdo/ubiquitous-journey/blob/main/values.yaml#L27) and no staging app definition in this file..but we have [test apps definition](https://github.com/springdo/ubiquitous-journey/blob/main/pet-battle/test/values.yaml) and no `test-app-of-pb` :D should we create `test-app-of-pb` instead of staging or was this intentional in order to not deploy PB? :D

Reply:
ahh I saw that wrong! staging one is disabled, sorry!